src/components/pokemonList/PokemonList.jsx

```diff
@@ -12,4 +13,46 @@ export function PokemonList({ pokemons }){
       })}
     </ul>
   );
-}
\ No newline at end of file
+}
```

Comment:
If you want to, you can remove the styling for the list items so it doesn't show on the page like this:
![image](https://user-images.githubusercontent.com/74560886/144903150-12bc2e35-b986-4fe3-af0a-1ae1990e6e30.png)